libnymea-core/hardware/network/mqtt/mqttproviderimplementation.h

```diff
@@ -64,7 +64,7 @@ private slots:
 private:
     MqttBroker* m_broker = nullptr;
 
-    QHash<QString, MqttChannel*> m_createdChannels;
+    QHash<QString, MqttChannel *> m_createdChannels;
```

Comment:
I suppose you can make this a MqttChannelImplementation, so you don't need to cast